gtk4/src/expression.rs

```diff
@@ -182,6 +246,46 @@ impl glib::value::ToValueOptional for Expression {
     }
 }
 
+// rustdoc-stripper-ignore-next
+/// Trait containing convenience methods in creating [`gtk::PropertyExpression`] that
+/// looks up a property of a [`glib::Object`].
+///
+/// # Example
+///
+/// `label_expression` is a [`gtk::Expression`] that looks up at Button's `label`
+/// property.
+///
+/// ```no_run
+/// # use gtk4 as gtk;
```

Comment:
While this is used elsewhere too, I think just using `use gtk4::prelude::*;` works better.

Reply:
No, we do the rename everywhere else